Applications/MaterialAssist/Client/MaterialAssistClientBoards.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System.Text;
+
+using HomagConnect.Base.Services;
+using HomagConnect.MaterialAssist.Contracts.Base;
+using HomagConnect.MaterialAssist.Contracts.Boards;
+using HomagConnect.MaterialAssist.Contracts.Boards.Interfaces;
+using HomagConnect.MaterialManager.Contracts.Material.Boards;
+
+namespace HomagConnect.MaterialAssist.Client
+{
+    /// <summary>
+    /// materialAssist client for boards
+    /// </summary>
+    public class MaterialAssistClientBoards : ServiceBase, IMaterialAssistClientBoards
+    {
+        /// <inheritdoc />
+        public MaterialAssistClientBoards(HttpClient client) : base(client) { }
+
+        #region Delete
+
+        /// <inheritdoc />
+        public async Task DeleteBoardEntity(string id)
+        {
+            var url = $"{_BaseRoute}/?{_Id}={Uri.EscapeDataString(id)}";
+
+            await DeleteObject(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task DeleteBoardEntities(IEnumerable<string> ids)
+        {
+            var url = $"{_BaseRoute}";
+
+            var boardCodes = new StringBuilder("?");
```

Comment:
Please have a look at StringExtensions.Join and the usage e.g. GetBoardTypeInventoryHistoryInternalAsync